practice_7.py

```diff
@@ -0,0 +1,11 @@
+l=[]
+x=int(input("enter the value of x="))
+y=int(input("enter the value of y="))
+for i in range(x):
+    a=[]
+    for j in range(y):
+        b=i*j
+        a.append(b)
+    l.append(a)
+for a in l:
```

Comment:
This will display each element in the list. The question just wants to display list l